### PR TITLE
[WIP] Add onnxruntime-extensions package

### PR DESCRIPTION
**Pull request by Chizkiyahu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Chizkiyahu** [comment]
@conda-forge-admin, please ping team conda-forge/help-python-c
install with `pip install .` on the local machine works ok 
but building with conda  fails 
thanks

File: recipes/onnxruntime-extensions/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - numpy >=1.18.5
```
Comment (xhochy) [inline recipes/onnxruntime-extensions/meta.yaml:23]: Please don't pin to use the global numpy pin
```suggestion
    - numpy
```

File: recipes/onnxruntime-extensions/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - ninja
+    - cmake
```
Comment (xhochy) [inline recipes/onnxruntime-extensions/meta.yaml:25]: These two should be in build

File: recipes/onnxruntime-extensions/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```
Comment (xhochy) [inline recipes/onnxruntime-extensions/meta.yaml:29]: As numpy has a run_exports, this is no longer necessary.
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by xhochy** [comment]
FYI @cbourjau

File: recipes/onnxruntime-extensions/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    - numpy
+    - pip
+  host:
+    - python
+    - setuptools
+    - wheel
+    - numpy
+    - pip
+  run:
+    - python
```
Comment (cbourjau) [inline recipes/onnxruntime-extensions/meta.yaml:34]: The library may also be used without `Python` (directly from C++), may it not?

**Pull request closed without merging**